src/style/text.rs

```diff
@@ -131,7 +131,7 @@ impl<'a, T: Into<FontDesc<'a>>> From<T> for TextStyle<'a> {
     fn from(font: T) -> Self {
         Self {
             font: font.into(),
-            color: BLACK.color(),
+            color: BLACK.to_backend_color(),
```

Comment:
One note I would add here is this change also improve the readability. Since `BLACK.color()` doesn't appear make sense literally.